src/pykeen/experiments/nodepiece/galkin2022_nodepiece_codexl_noancs.yaml

```diff
@@ -0,0 +1,33 @@
+metadata:
+  title: Learn CoDEx-Large Dataset with NodePiece + RotatE (no anchor tokenization) as described by Galkin et al., 2022
+pipeline:
+  dataset: CoDExLarge
+  evaluator_kwargs:
+    filtered: true
+  loss: BCELoss
+  model: NodePiece
+  model_kwargs:
+    entity_initializer: xavier_uniform_
+    relation_initializer: init_phases
+    relation_constrainer: complex_normalize
+    embedding_dim: 200   # for both anchors and relations
+    interaction: RotatEInteraction
+    aggregation: mlp
+    tokenizers: RelationTokenizer
+    num_tokens: 6  # 6 relations
+    tokenizers_kwargs: [{},{}]  # no params for RelationTokenizer
```

Comment:
superfluous kwargs? there is only a single tokenizer

Reply:
I kept in mind the `make_many` function from class_resolver which needs a list of kwargs per tokenizer and selects a relevant list in the order. If it works w/o the list of empty dicts - let's purge it 🔥